scripts/lb-wrapper

```diff
@@ -6,6 +6,10 @@ URL_OR_SEARCH_TERM="$2"
 LOG_FILE="/var/log/xklb.log"
 XKLB_DB_FILE="/library/calibre-web/xklb-metadata.db"
 TMP_DOWNLOADS_DIR="/library/downloads/calibre-web"
+# outtmpl explanation:
+# https://github.com/yt-dlp/yt-dlp#output-template
```

Comment:
Include both code snippets?

```suggestion
# https://github.com/yt-dlp/yt-dlp#output-template
# https://github.com/chapmanjacobd/library/blob/e682ac20f5dcc4fa94238b57931f4705b9515059/xklb/createdb/tube_backend.py#L385-L388
```

Reply:
I already included both comments per https://github.com/iiab/calibre-web/pull/177/commits/397b8c2283547fbdc857efa74b44d4fd08a4e9d0

Reply:
Both code snippets is my request:

#L385-L388 and #L508-L523

Reply:
Ok, I see